src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/compile.py

```diff
@@ -13,8 +13,8 @@
 from torch.fx import GraphModule
```

Comment:
Don't we need to have separate file for executorch implementation? Just to avoid mix with `torch.compile`

Reply:
Currently the only difference in "openvino_compile" for toch.compile and executorch is the decoder type it uses (We could add disabling caching for executorch based on the comment above). Otherwise, the functionality is same for both. Since the difference is minimal, the method is not exposed to users and only used internally, we wanted to use common "openvino_compile" for both cases in the initial implementation.